### PR TITLE
assignment 4 - Team Member Cards

### PR DESCRIPTION
**Pull request by aditifarkya**
Description - 
Created team card component to display image, name , experience and linked-in url of team members.

Functionality After:-
assignment4 1
assignment4

### CODE REVIEW

File: src/components/card-component/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import "./style.css";
+const CardComponent = (props) => {
```
Comment (ap221882) [inline src/components/card-component/index.js:3]: You can add a new line after the imports for better code readability.

File: src/index.js [hunk 2]
```diff
@@ -1,33 +1,59 @@
...
+import teamdata from "./components/card-component/config";
 
-//create functional nested component
-const name = "Third";
-const SecondHeadingComponent = () => {
-  return <h2>Second Heading</h2>;
+const CardContainer = () => {
+  // using for loop only prints first object of teamdata array
+  // for (let i = 0; i <= teamData.length; i++) {
+  //   console.log(i);
```
Comment (ap221882) [inline src/index.js:11]: Try to return an array of JSX elements by the for loop

**Review by ap221882 — COMMENTED** [review]
Added some suggestions. Nice styling.

File: src/index.js (hunk 2 above)
Comment (aditifarkya) [reply src/index.js:11]: yeah i tried that by pushing it to an array and it worked.

**Pull request merged**